constraints/cv32e40p_core.sdc

```diff
@@ -0,0 +1,167 @@
+#
+# Use of this file means you agree to the terms and conditions
+# of the license and are in full compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+#     https://solderpad.org/licenses/SHL-2.0/
+#
+# Unless required by applicable law or agreed to in writing, software
+# and hardware implementations thereof distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESSED OR IMPLIED.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#//////////////////////////////////////////////////////////////////////////////
+# Engineer:       Arjan Bink - arjan.bink@silabs.com                         //
+#                                                                            //
+# Project Name:   CV32E40P                                                   //
+#                                                                            //
+# Description:    Example synthesis constraints.                             //
+#                                                                            //
+#                 The clock period and input/output delays are technology    //
+#                 and project dependent and are expected to be adjusted as   //
+#                 needed.                                                    //
+#                                                                            //
+#                 OBI related bus inputs arrive late on purpose and OBI      //
+#                 related outputs are available earlier (as they shall not   //
+#                 combinatorially depend o nthe OBI inputs)                  //
+#                                                                            //
+#//////////////////////////////////////////////////////////////////////////////
+
+# 200MHz
+set clock_period 5.0
+
+# Input delays for interrupts
+set in_delay_irq          [expr $clock_period * 0.50] 
+# Output delays for interrupt related signals
+set out_delay_irq         [expr $clock_period * 0.25] 
+
+# Input delays for early signals
+set in_delay_early [expr $clock_period * 0.10] 
+
+# OBI inputs delays
+set in_delay_instr_gnt    [expr $clock_period * 0.83]
+set in_delay_instr_rvalid [expr $clock_period * 0.83]
+set in_delay_instr_rdata  [expr $clock_period * 0.83]
+
+set in_delay_data_gnt     [expr $clock_period * 0.83]
+set in_delay_data_rvalid  [expr $clock_period * 0.83]
+set in_delay_data_rdata   [expr $clock_period * 0.83]
+
+# OBI outputs delays
+set out_delay_instr_req  [expr $clock_period * 0.57]
+set out_delay_instr_addr [expr $clock_period * 0.57]
+
+set out_delay_data_req   [expr $clock_period * 0.57]
+set out_delay_data_we    [expr $clock_period * 0.57]
+set out_delay_data_be    [expr $clock_period * 0.57]
+set out_delay_data_addr  [expr $clock_period * 0.57]
+set out_delay_data_wdata [expr $clock_period * 0.57]
```

Comment:
Why cannot we assume that output and input delay from/to memories are the same?
These numbers (0.83 and 0.57) look very specific to a given technology

Reply:
Hi @davideschiavone I want to convey that memory related output come out roughly midway through the cycle and that memory related inputs are expected to arrive late. The numbers will always be tech specific, which is why there is a comment at the top about that. Would it be okay if I just round the numbers: 0.57 -> 0.6, 0.83 -> 0.8. I do not really care about the actual numbers as long as the signals arriving from memory are assumed to arrive later than that we will guarantee the availability of the signals towards memory (this will assure there are no dis-allowed combinatorial paths inside the core for the OBI interfaces. If you have other nrs in mind, please let me know.

Reply:
0.6 and 0.8 with the comments look ok. 
Is there a specific reason why we are saying that we leave inside the core 40% to go out (1-0.6) and we have only 20% (1-0.8) from the response? why not , for example, 30% each?

Reply:
The numbers are based on what is possible plus what is expected from the environment. In particular the OBI inputs are expected to arrive very late in the cycle if you are building a system without wait states.

Reply:
Yes I understand the number per se, I don't understand why it's different between input and output